src/bundle/Resources/views/content/content_edit.html.twig

```diff
@@ -13,9 +13,18 @@
 
 {% block javascripts %}
     {%  javascripts
+        '@EzPlatformAdminUiAssetsBundle/Resources/public/vendors/flatpickr/dist/flatpickr.js'
         '@EzPlatformAdminUiBundle/Resources/public/js/scripts/fieldType/base/base-field.js'
         '@EzPlatformAdminUiBundle/Resources/public/js/scripts/fieldType/*'
     %}
         <script type="text/javascript" src="{{ asset_url }}"></script>
     {% endjavascripts %}
 {% endblock %}
+
+{% block stylesheets %}
+    {%  stylesheets
+        'bundles/ezplatformadminuiassets/vendors/flatpickr/dist/flatpickr.min.css'
```

Comment:
`@EzPlatformAdminUiAssetsBundle/Resources/public/vendors/flatpickr/dist/flatpickr.min.css`? I never know which one we should use.

Reply:
Corrected